// 这里我们用的是proto3版本，版本号是一定要指定的哦！
syntax = "proto3";

import "google/protobuf/wrappers.proto";

//这是供java端调用时用到的，咱们python的不用管
//option java_package = "xx.xx.core.grpc.lib.label";
//option java_multiple_files = true;

//package label;

service LabelService {
  /**
  样本集接口
   */
  rpc receiveSample(SampleRequestList) returns (google.protobuf.BoolValue) {}
}
message SampleRequestList {
  repeated SampleRequest reqList = 1;
}

message SampleRequest {
  /**
  标签值
   */
  string labelValue = 1;
  /**
  样本集数据
   */
  string text = 2;
}
